db/migrate/20230623063944_create_favorite_chefs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFavoriteChefs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :favorite_chefs do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :chef, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
```

Comment:
この中間テーブルではuser_idカラムとchef_idカラムの組み合わせは1パターンしか許容していないため、以下のようなユニーク制約をつけてみて下さい！

```suggestion
      t.timestamps
    end
    
    add_index :favorite_chefs, [:user_id, :chef_id], unique: true
  end
```

Reply:
修正しました！